src/main/java/com/wfector/command/Help.java

```diff
@@ -2,48 +2,61 @@
-            helpItems.add("&7 /csn &dclear &f- Remove read sales");
+        if(sender.hasPermission("csn.command.history") && plugin.getMessage("help.history") != null) {
+            helpItems.add(plugin.getMessage("help.history"));
         }
-        helpItems.add("&c ");
-
-        if (sender.hasPermission("csn.command.history.others")) {
-            helpItems.add("&7 /csn &dhistory <username> [<page>]&f- View sales of another player");
+        if(sender.hasPermission("csn.command.read") && plugin.getMessage("help.read") != null) {
+            helpItems.add(plugin.getMessage("help.read"));
         }
-        if (sender.hasPermission("csn.command.cleandatabase")) {
-            helpItems.add("&7 /csn &ccleandatabase &f- Remove database entries. Parameters:");
-            helpItems.add("&c --older-than, -o <days> &f- Removes entries older than <days>");
-            helpItems.add("&c --user, -user <username/uuid> &f- Removes entries from a single user only");
-            helpItems.add("&c --read-only, -r &f- Removes only read entries, default behaviour");
-            helpItems.add("&c --all, -a &f- Removes all entries");
+        if(sender.hasPermission("csn.command.clear") && plugin.getMessage("help.clear") != null) {
+            helpItems.add(plugin.getMessage("help.clear"));
         }
-        if (sender.hasPermission("csn.command.upload")) {
-            helpItems.add("&7 /csn &cupload &f- Force update databases");
+        if(sender.hasPermission("csn.command.history.others") && plugin.getMessage("help.history-others") != null) {
+            helpItems.add(plugin.getMessage("help.history-others"));
         }
-        if (sender.hasPermission("csn.command.convert")) {
-            helpItems.add("&7 /csn &cconvert &f- Convert database to UUIDs");
+        if(sender.hasPermission("csn.command.cleandatabase") && plugin.getMessage("help.cleandatabase") != null) {
+            helpItems.add(plugin.getMessage("help.cleandatabase"));
         }
-
-        if (sender.hasPermission("csn.command.reload")) {
-            helpItems.add("&7 /csn &creload &f- Reload configuration");
+        if(sender.hasPermission("csn.command.cleandatabase") && plugin.getMessage("help.cleandatabase-older-than") != null) {
+            helpItems.add(plugin.getMessage("help.cleandatabase-older-than"));
+        }
+        if(sender.hasPermission("csn.command.cleandatabase") && plugin.getMessage("help.cleandatabase-user") != null) {
+            helpItems.add(plugin.getMessage("help.cleandatabase-user"));
+        }
+        if(sender.hasPermission("csn.command.cleandatabase") && plugin.getMessage("help.cleandatabase-read-only") != null) {
+            helpItems.add(plugin.getMessage("help.cleandatabase-read-only"));
+        }
+        if(sender.hasPermission("csn.command.cleandatabase") && plugin.getMessage("help.cleandatabase-all") != null) {
+            helpItems.add(plugin.getMessage("help.cleandatabase-all"));
+        }
```

Comment:
Would be good to merge all these so that `sender.hasPermission("csn.command.cleandatabase")` is only called once?